packages/paste-website/src/pages/patterns/delete/index.mdx

```diff
@@ -97,28 +97,40 @@ import { Toast } from "@twilio-paste/core/toast";
 
 ### General
 
-The delete pattern should be utilized when a user is destroying an object. It is critical to communicate delete actions to our users in a consistent way, considering deletions come with significant consequences and often cannot be undone.
+The delete pattern should be used when a user is destroying an object. It is critical to communicate delete actions to our users in a consistent way, considering deletions come with significant consequences and often cannot be undone.
 
 The delete action should:
 
-- Be triggered by a destructive Button.
-- Generally be placed at the bottom of an [Object Details](/patterns/object-details) page.
-- Not usually be the primary action on the page. There are many variants of differing visual prominence to choose from to achieve the correct hierarchy for the delete action trigger.
-- Follow certain guidelines based on the [severity](#variations) of the deletion.
+- Be initiated by a destructive Button.
+- Rarely be the primary action on the page. There are many variants of differing visual prominence to choose from to achieve the correct hierarchy for the delete action trigger.
+- Follow guidelines based on the [severity](#variations) of the deletion.
+
+Medium and high severity delete confirmations are displayed using an [Alert Dialog[(/components/alert-dialog)], since it requires the user to interact with the component.
+
+## Composition
+
+Delete messages should follow the [Confirmation pattern](/patterns/confirmation). 
+
+Avoid using negative contractions like can’t or won’t, since user comprehension can suffer in stressful scenarios.
 
 ## Variations
 
-### Low-severity
+### Severity
```

Comment:
suggestion: replace the "Variations" heading with "Severity" so that all the subsequent headings can move back up in hiearchy

Reply:
done